java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -4146,6 +4146,102 @@ public void xgroupCreateConsumer_xgroupDelConsumer_xreadgroup_xack(BaseClient cl
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xgroupSetId_entriesRead(BaseClient client) {
+        String key = "testKey" + UUID.randomUUID();
+        String nonExistingKey = "group" + UUID.randomUUID();
+        String stringKey = "testKey" + UUID.randomUUID();
+        String groupName = UUID.randomUUID().toString();
+        String consumerName = UUID.randomUUID().toString();
+        String streamId0 = "0";
+        String streamId1_0 = "1-0";
+        String streamId1_1 = "1-1";
+        String streamId1_2 = "1-2";
+
+        // Setup: Create stream with 3 entries, create consumer group, read entries to add them to the
+        // Pending Entries
+        // List.
```

Comment:
```suggestion
        // Pending Entries List.
```
Spotless plays with you!